airflow/configuration.py

```diff
@@ -76,8 +77,8 @@ def run_command(command):
     Runs command and returns stdout
     """
     process = subprocess.Popen(
-        command.split(), stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-    output, stderr = process.communicate()
+        shlex.split(command), stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    output, stderr = [stream.decode() for stream in process.communicate()]
```

Comment:
Argh, I forgot that the default encoding is ascii in python2.  I think it should be reasonable to always decode as unicode, though.